pandas/core/generic.py

```diff
@@ -7576,10 +7598,10 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
         specify on level the resampling needs to take place.
 
         >>> time = pd.date_range('1/1/2000', periods=5, freq='T')
-        >>> df2 = pd.DataFrame(data=10*[range(4)],
-                               columns=['a', 'b', 'c', 'd'],
-                               index=pd.MultiIndex.from_product([time, [1, 2]])
-                               )
+        >>> df2 = pd.DataFrame(data=10 * [range(4)],
+        ...                    columns=['a', 'b', 'c', 'd'],
+        ...                    index=pd.MultiIndex.from_product([time, [1, 2]])
+        ...                    )
         >>> df2.resample('3T', level=0).sum()
                              a  b   c   d
         2000-01-01 00:00:00  0  6  12  18
```

Comment:
The `See Also` section needs a capital `A` in the title, and the section should go before the examples, not after.